docs/index.rst

```diff
@@ -116,15 +111,14 @@ On the left-hand side, from top to bottom, the tabs are:
 - :ref:`SQL console <sql-console>`
 - :ref:`Tables browser <tables-browser>`
 - :ref:`Views browser <views-browser>`
-- :ref:`Shards browser <shards-browser>` (`enterprise feature`_)
+- :ref:`Shards browser <shards-browser>`
 - :ref:`Cluster browser <cluster-browser>`
-- :ref:`Monitoring overview <monitoring-overview>` (`enterprise feature`_)
-- :ref:`Privileges browser <privileges-browser>` (`enterprise feature`_)
+- :ref:`Monitoring overview <monitoring-overview>`
+- :ref:`Privileges browser <privileges-browser>`
 - :ref:`Help screen <help-screen>`
 
 
 .. _Cluster name: https://crate.io/docs/crate/reference/en/latest/config/node.html#basics
 .. _Community Edition: https://crate.io/docs/crate/reference/en/latest/editions.html#cratedb-community-edition
```

Comment:
is this still used?

Reply:
You mean the _monitoring_ and the _privileges_ views? Sure, both are two of the plugins which just have been available in the former enterprise build, see [1].

[1] https://github.com/crate/crate-admin/tree/master/app/plugins

Reply:
> You mean the monitoring and the privileges views? 

No I meant the community edition reference ;) 
`.. _Community Edition: https://crate.io/docs/crate/reference/en/latest/editions.html#cratedb-community-edition`

Reply:
Removed. Thanks!